interview-questions.md

```diff
@@ -5,64 +5,88 @@ First, without external resources. Challenge yourself to answer from memory as i
 
 Then, research the question to expand on your answer. Even if you feel you have answered the question completely, there is always something more to learn. Write your researched answer in your OWN WORDS.
 
+
+// 
+
+I wrote all this below in the example, then saw it was an example. 
+there was nothing there, so i guess i just did an extra question. 
+which just helps me so im happy about it for practice im leaving it for myself
+
+//
+
 INSTRUCTOR EXAMPLE: What is a conditional statement?
 
-  Your answer:
+  Your answer: A conditional statement is a list of computer command instuctions, that require something to happen for the code to take action. They use if or else statements to input data text strings or numeric values. This requires the code to act on a contition put in by the programmer to get a specific result.
 
-  Researched answer:
+  Researched answer: Decision structures, also called decision trees, conditional statements, or if/else statements, are fundamental to computer programming. Conditional statements are a sequence of well-defined instructions that produce a unique output based on the value of the input. Conditionals follow a flowchart-like structure and allow you create logic in your code.
 
 
 
 1. What is git? What is the difference between git and Github?
 
-  Your answer:
+  Your answer: Git is a command that allows you to connect, create branches, make changes,pull files, and publish work. This command works with Github from the terminal taking files from local to online. Github is a file sharing website where developers share code and work together on projects it can also work as reasume with a catalog of all your work. 
+
+  Researched answer: Git is a free, open-sourced version control system widely used in the programming world. It is a very powerful tool used for sharing and working on code and allows for many people to contribute to the same code base. Git was created in 2005 by Linus Torvalds, the creator of Linux, to track changes in source code. Git is accessed primarily through the command line.
+
+History of the name git according to Linus is quite funny. 'Git' is British slang for pig headed or argumentative. This will feel very appropriate at moments when interacting with git. Torvolds says, while 'git' was a random three-letter combination that is pronounceable, and not actually used by any common UNIX command, it can change meaning, depending on your mood:
+
+Global Information Tracker: if you're in a good mood, and it actually works for you. Angels sing and light suddenly fills the room
+Goddamn Idiotic Truckload of sh*t: when it breaks.
```

Comment:
🤣🤣🤣  Love this!!!!!!  Excellent researched answer!!!

Reply:
i need to change this and i will but it is very accurate